test/e2e/apimachinery/webhook.go

```diff
@@ -1947,8 +1960,15 @@ func registerSlowWebhook(f *framework.Framework, configName string, context *cer
 	})
 	framework.ExpectNoError(err, "registering slow webhook config %s with namespace %s", configName, namespace)
 
-	// The webhook configuration is honored in 10s.
-	time.Sleep(10 * time.Second)
+	// TODO: Remove this and instead register the slow webhook only once and then updated it and wait
+	// for the transitions to occur, which are all possible to detect
+	if *policy == admissionregistrationv1.Fail && timeout != nil && *timeout <= 5 {
+		err = waitAlwaysAllowDelayWebhookReady(f, client)
+		framework.ExpectNoError(err, "waiting for always allow config map with delay webhook to be ready")
+	} else {
+		// Since there is no client visible change, wait 10s for the webhook configuration to be honored.
```

Comment:
for failurePolicy:Ignore, if a request completes faster than the slow webhook timeout, we know it's not effective yet, right?